assets/sass/components/key-metrics/_googlesitekit-key-metrics-setup-cta.scss

```diff
@@ -119,7 +119,7 @@
 
 			@media (min-width: $bp-nonMobile) {
 				display: flex;
-				gap: 20px;
+				gap: 4px;
```

Comment:
Same here, why the gap so different now?

Reply:
This also uses the same concept as above.

The key metrics setup CTA widget applies a gap of `20px` between the two actions. Since the tertiary button already has a horizontal padding of `16px`, we change the `gap` to `4px` so that the visual white space remains the same.

Before:
![image](https://github.com/google/site-kit-wp/assets/20284937/41a1a576-d34f-470d-bb61-2984f4cc6f8b)

After:
![image](https://github.com/google/site-kit-wp/assets/20284937/5ccdf375-2478-4d0d-932d-116165a02901)

After (tertiary button hovered):
![image](https://github.com/google/site-kit-wp/assets/20284937/0dd28d91-b7fe-47e4-9a39-9409c03249d2)